get.sh

```diff
@@ -73,21 +73,32 @@ if [ -z "$CODACY_REPORTER_VERSION" ]; then
     CODACY_REPORTER_VERSION="latest"
 fi
 
-download_url() {
-    grep browser_download_url | grep $1 | cut -d '"' -f 4
-}
-
-download_using_wget_or_curl() {
-    api_url="https://api.github.com/repos/codacy/codacy-coverage-reporter/releases/$CODACY_REPORTER_VERSION"
+curl_or_wget() {
     if [ -x "$(which curl)" ]; then
-        curl -# -LS -o "$codacy_reporter" "$(curl -LSs $api_url | download_url $1)"
+        echo "curl -LSs"
     elif [ -x "$(which wget)" ] ; then
-        wget -O "$codacy_reporter" "$(wget -O - $api_url | download_url $1)"
+        echo "wget -O"
     else
         fatal "Could not find curl or wget, please install one."
     fi
 }
 
+get_version() {
+    if [ "$CODACY_REPORTER_VERSION" == "latest" ]; then
+        bintray_latest_api_url="https://api.bintray.com/packages/codacy/Binaries/codacy-coverage-reporter/versions/_latest"
+        latest="$($(curl_or_wget) $bintray_latest_api_url | sed -e 's/.*name.*\([0-9]\+[.][0-9]\+[.][0-9]\+\).*/\1/')"
```

Comment:
after the refactor this line should be enough
```suggestion
        latest="$(download $bintray_latest_api_url | sed -e 's/.*name.*\([0-9]\+[.][0-9]\+[.][0-9]\+\).*/\1/')"
```